knimin/lib/data_access.py

```diff
@@ -2399,6 +2399,178 @@ def getProjectNames(self):
         sql = """SELECT project FROM project"""
         return [x[0] for x in self._con.execute_fetchall(sql)]
 
+    def create_sample(self, sample_ids):
```

Comment:
What if the sample already exists? The DB will raise an error, but it may be useful to define exactly which is the behavior that we want to happen.

Also, the samples will always belong to a project. How do you associate them to a project?